feat: created the footer

ductrl opened the pull request:
## What does this PR do?

Fixes #43 

## Type of change

<!-- Please delete bullets that are not relevant. -->
- New feature (non-breaking change which adds functionality)

## Mandatory Tasks

- [X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

<!-- Please remove all the irrelevant bullets to your PR -->

- I have read the [contributing guide](https://github.com/DhanrajKamble/ThePokedex/blob/master/CONTRIBUTING.MD)
- My code follow the style guidelines of this project
- I commented my code, particularly in hard-to-understand areas
- I haved checked if my PR needs changes to the documentation
- I have checked if my changes generate no new warnings

ductrl committed:
Created the footer

Rahufkhan commented on style.css:
```diff
@@ -1,299 +1,328 @@
-body{
-    background-color:#F0F0E5;
+body {
+  background-color: #f0f0e5;
```
don't change the code #F0F0E5 to #f0f0e5 both one and some leave as it is

Rahufkhan commented on style.css:
```diff
@@ -1,299 +1,328 @@
+
+#footer-logo {
+  /* Setting the size for the logo */
+  width: 10vw;
```
fix logo width in responsive mobile screen size. logo is decreasing its width wen we shift to mobile size

Rahufkhan commented:
don't change the previous code of css just add the footer css and footer responsive css to it.

Rahufkhan commented:
increase logo size to mobile screen size

ductrl committed:
Update style.css based on feedbacks

I really appreciate the feedbacks. I'm really sorry for the color code formatting, I use the prettier extension in VS Code so all the color codes are lowercased automatically.

Rahufkhan commented:
in mobile size screen footer font size should be like this.

![mobile_size_footer](https://github.com/DhanrajKamble/ThePokedex/assets/62764275/9566df2c-27e5-444c-b795-5253db7c7862)

Rahufkhan commented on index.html:
```diff
@@ -288,7 +288,12 @@ <h1>Fun and Tools</h1><hr>
-        <p>All ©️copyrights reserved ThePokedex 2023...</p> 
+        <!-- Linking the site to the logo -->
+        <a href="https://the-pokedex-nu.vercel.app/">
+            <img src="https://user-images.githubusercontent.com/62764275/282056861-444edccd-a670-421a-a1f0-45caed15582d.svg" alt="GO Hub Logo" id="footer-logo">
```
you can upload footer logo image file and upload it in the images folder by downloading it in the issue #43

ductrl committed:
Added settings for live server testing and gohub logo, will remove the settings later

ductrl committed:
Merge branch 'create-footer' of https://github.com/ductrl/ThePokedex into create-footer

ductrl committed:
Added logo to images, fixed footer css for mobile based on feedbacks

ductrl commented:
I updated the files based on your feedbacks. I'm pretty unexperienced in coding so I'd love to hear more feedbacks from you!

Rahufkhan commented:
No worries, bro, that's what you learn from your mistake. Just keep it up. You have done great work; I will merge it. After all it is a beginner-friendly project.

ductrl commented:
Thank you so much for the kind words. Please let me know if I can help with anything in the future. Wish you the best of luck with the project!
